apps/perms/api/user_permission/nodes.py

```diff
@@ -18,7 +19,7 @@
 ]
 
 
-class BaseUserPermedNodesApi(SelfOrPKUserMixin, ListAPIView):
+class BaseUserPermedNodesApi(SelfOrPKUserMixin, RebuildTreeMixin, ListAPIView):
     serializer_class = serializers.NodePermedSerializer
 
     def get_queryset(self):
```

Comment:
The code appears to be written in Python with Django. There seem to have no syntax errors detected, but I would like more specific information on how the functions or classes are being used and their interactions within the project for a detailed analysis.

However, if there is anything you want specific changes, suggestions for improvements, or need debugging/development assistance related to this codebase, feel free to describe your concern further!